.gitignore

```diff
@@ -2,6 +2,7 @@ Output/
 data/
 logs/
 .idea/
+error_file.csv
```

Comment:
This commit only added this one change in .gitignore?

Reply:
Urm ... whooops ... good spot, please see the next commit ...